src/main/resources/turtychemistry.mixins.json

```diff
@@ -1,10 +1,11 @@
 {
 	"required": true,
-	"package": "io.github.darealturtywurty.turtychemistry.mixins",
+	"package": "io.github.darealturtywurty.turtychemistry.mixin",
```

Comment:
Is there a reason you changed it to `mixin`? `mixins` is the standard and should generally be used over `mixin`.

Reply:
this change is weird, I didn't change it to mixin, i'll just quickly fix that